[2022-12-09 10:41:25,757] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_solar.transform.preprocess_weather scheduled__2022-12-08T00:00:00+00:00 [queued]>
[2022-12-09 10:41:25,818] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_solar.transform.preprocess_weather scheduled__2022-12-08T00:00:00+00:00 [queued]>
[2022-12-09 10:41:25,820] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-09 10:41:25,822] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-09 10:41:25,823] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-09 10:41:25,904] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): transform.preprocess_weather> on 2022-12-08 00:00:00+00:00
[2022-12-09 10:41:25,922] {standard_task_runner.py:52} INFO - Started process 177 to run task
[2022-12-09 10:41:25,937] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_solar', 'transform.preprocess_weather', 'scheduled__2022-12-08T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/dag_preprocess.py', '--cfg-path', '/tmp/tmpwmxk8j7q', '--error-file', '/tmp/tmp02jobzcm']
[2022-12-09 10:41:25,943] {standard_task_runner.py:80} INFO - Job 35: Subtask transform.preprocess_weather
[2022-12-09 10:41:26,010] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:470: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2022-12-09 10:41:26,234] {task_command.py:369} INFO - Running <TaskInstance: etl_solar.transform.preprocess_weather scheduled__2022-12-08T00:00:00+00:00 [running]> on host 4d9af2ba34e6
[2022-12-09 10:41:26,661] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_solar
AIRFLOW_CTX_TASK_ID=transform.preprocess_weather
AIRFLOW_CTX_EXECUTION_DATE=2022-12-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-08T00:00:00+00:00
[2022-12-09 10:41:26,780] {python.py:173} INFO - Done. Returned value was: None
[2022-12-09 10:41:26,919] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_solar, task_id=transform.preprocess_weather, execution_date=20221208T000000, start_date=20221209T104125, end_date=20221209T104126
[2022-12-09 10:41:27,014] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-09 10:41:27,164] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-12-09 11:20:55,985] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_solar.transform.preprocess_weather scheduled__2022-12-08T00:00:00+00:00 [queued]>
[2022-12-09 11:20:56,045] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_solar.transform.preprocess_weather scheduled__2022-12-08T00:00:00+00:00 [queued]>
[2022-12-09 11:20:56,047] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-09 11:20:56,048] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-09 11:20:56,050] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-09 11:20:56,122] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): transform.preprocess_weather> on 2022-12-08 00:00:00+00:00
[2022-12-09 11:20:56,138] {standard_task_runner.py:52} INFO - Started process 276 to run task
[2022-12-09 11:20:56,153] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_solar', 'transform.preprocess_weather', 'scheduled__2022-12-08T00:00:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/dag_preprocess.py', '--cfg-path', '/tmp/tmpw7r3lrit', '--error-file', '/tmp/tmp6sm0qi6g']
[2022-12-09 11:20:56,158] {standard_task_runner.py:80} INFO - Job 60: Subtask transform.preprocess_weather
[2022-12-09 11:20:56,228] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:470: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2022-12-09 11:20:56,475] {task_command.py:369} INFO - Running <TaskInstance: etl_solar.transform.preprocess_weather scheduled__2022-12-08T00:00:00+00:00 [running]> on host 4d9af2ba34e6
[2022-12-09 11:20:56,907] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_solar
AIRFLOW_CTX_TASK_ID=transform.preprocess_weather
AIRFLOW_CTX_EXECUTION_DATE=2022-12-08T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-08T00:00:00+00:00
[2022-12-09 11:20:57,026] {python.py:173} INFO - Done. Returned value was: None
[2022-12-09 11:20:57,087] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_solar, task_id=transform.preprocess_weather, execution_date=20221208T000000, start_date=20221209T112055, end_date=20221209T112057
[2022-12-09 11:20:57,173] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-09 11:20:57,431] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
